protocol-concepts/governance/verification.md

```diff
@@ -12,7 +12,21 @@ Rate Feed identifiers are the [report targets for oracles](../../developers/smar
+<table><thead><tr><th width="143.33333333333331">RateFeed</th><th width="295">ID</th><th>Explanation</th></tr></thead><tbody>
+<tr><td>CELO/USD</td><td>0x765DE816845861e75A25fCA122bb6898B8B1282a</td><td>The <a href="https://explorer.celo.org/mainnet/address/0x765DE816845861e75A25fCA122bb6898B8B1282a">cUSD contract address</a></td></tr>
+<tr><td>CELO/EUR</td><td>0xD8763CBa276a3738E6DE85b4b3bF5FDed6D6cA73</td><td>The <a href="https://explorer.celo.org/mainnet/address/0xD8763CBa276a3738E6DE85b4b3bF5FDed6D6cA73">cEUR contract address</a></td></tr>
+<tr><td>CELO/BRL</td><td>0xe8537a3d056DA446677B9E9d6c5dB704EaAb4787</td><td>The <a href="https://explorer.celo.org/mainnet/address/0xe8537a3d056DA446677B9E9d6c5dB704EaAb4787">cREAL contract address</a></td></tr>
+<tr><td>CELO/XOF</td><td>0x73F93dcc49cB8A239e2032663e9475dd5ef29A08</td><td>The <a href="https://explorer.celo.org/mainnet/address/0x73F93dcc49cB8A239e2032663e9475dd5ef29A08">eXOF contract address </a></td></tr>
+<tr><td>CELO/KES</td><td>0x456a3D042C0DbD3db53D5489e98dFb038553B0d0</td><td>The <a href="https://explorer.celo.org/mainnet/address/0x456a3D042C0DbD3db53D5489e98dFb038553B0d0">cKES contract address </a></td></tr>
+<tr><td>USDC/USD</td><td>0xA1A8003936862E7a15092A91898D69fa8bCE290c</td><td><code>address(uint160(uint256(keccak256("USDCUSD"))))</code></td>
+</tr><tr><td>USDC/EUR</td><td>0x206B25Ea01E188Ee243131aFdE526bA6E131a016</td><td><code>address(uint160(uint256(keccak256("USDCEUR"))))</code></td></tr>
+<tr><td>USDC/BRL</td><td>0x25F21A1f97607Edf6852339fad709728cffb9a9d</td><td><code>address(uint160(uint256(keccak256("USDCBRL"))))</code></td></tr>
+<tr><td>EUROC/EUR</td><td>0x26076B9702885d475ac8c3dB3Bd9F250Dc5A318B</td><td><code>address(uint160(uint256(keccak256("EUROCEUR"))))</code></td></tr>
+<tr><td>EUROC/XOF</td><td>0xed35e46b095197da30ddffa5b91d386886d5ce0d</td><td><code>address(uint160(uint256(keccak256("EUROCXOF"))))</code></td></tr>
+<tr><td>EUR/XOF</td><td>0x40dc8528167557353fdcd98548ab2139a670dd0b</td><td><code>address(uint160(uint256(keccak256("EURXOF"))))</code></td></tr>
+<tr><td>KES/USD</td><td>0xbAcEE37d31b9f022Ef5d232B9fD53F05a531c169</td><td><code>address(uint160(uint256(keccak256("USDTUSD"))))</code></td></tr>
```

Comment:
I think these are the wrong way around? When I preview I see the `address(uint160(uint256(keccak256("USDTUSD"))))` next to KES/USD